### PR TITLE
Add 2.1.0 notes

### PR DESCRIPTION
**Pull request by jwaimann**
Reviewers: @patricioe @dendril

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,5 +1,12 @@
 CHANGES
 
+2.1.0
+- Localmode to support reload features.
+- Refactor InMemory Metrics and Impressions to comply cache interface style.
+- Support for Redis cache and Redis client.
+- Configuration to support multiple modes and Redis parameters.
+- Support for multiple splits evaluation by including GetTreatments method.
+
```
Comment (patricioe) [inline CHANGES.txt]: extra +

Comment (patricioe) [inline CHANGES.txt]: also, any backward incompatible change?

File: NEWS.txt [hunk 2]
```diff
@@ -1,5 +1,12 @@
 NEWS
 
+2.1.0
+
+- Now supporting Redis as a cache adapter
```
Comment (patricioe) [inline NEWS.txt]: Please don't repeat the notes in CHANGES. This is mostly for code changes needed to make the new version work with their existing code.

File: CHANGES.txt (hunk 1 above)
Comment (jwaimann) [inline CHANGES.txt]: The only added option is "Mode", and its default is ´Standalone´. So user don't need to make it explicit. (no incompatible change)

File: NEWS.txt (hunk 2 above)
Comment (jwaimann) [inline NEWS.txt]: ok, got it

File: NEWS.txt [hunk 3]
```diff
@@ -2,10 +2,7 @@ NEWS
 
 2.1.0
 
-- Now supporting Redis as a cache adapter
-- Local mode supports reload features
 - Configuration options for Mode and Redis parameters added.
```
Comment (jwaimann) [inline NEWS.txt]: This change is not incompatible, because we have default values. should I remove it too and leave it as No news?

**Pull request merged**